packages/beeq/src/components/slider/bq-slider.tsx

```diff
@@ -173,7 +173,9 @@ export class BqSlider {
   private runUpdates = () => {
     this.updateProgressTrack();
     this.syncInputsValue();
-    this.setThumbPosition();
+    requestAnimationFrame(() => {
+      this.setThumbPosition();
+    });
```

Comment:
Can we add an empty line in between?

```suggestion
    
    requestAnimationFrame(() => {
      this.setThumbPosition();
    });
```